src/main/java/marquez/client/models/StreamMeta.java

```diff
@@ -0,0 +1,35 @@
+package marquez.client.models;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import javax.annotation.Nullable;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.experimental.SuperBuilder;
+import marquez.client.utils.JsonUtils;
+
+@SuperBuilder
+@JsonPropertyOrder({
```

Comment:
Needed to bring this in to ensure string equality. schemaLocation was being added after description.